src/Math-Statistics/Collection.extension.st

```diff
@@ -0,0 +1,37 @@
+Extension { #name : #Collection }
+
+{ #category : #'*Math-Statistics' }
```

Comment:
@SergeStinckwich I suggested that geometric and harmonic mean computation should be in our Statistics package after some quick research on Wikipedia. Would you agree? Are there alternatives?

Reply:
Note that there is an arithmetic-geometric mean already implemented in ArbitraryPrecisionFloat.
This is absolutely not specific to statistics package.
Eventually, for small utilities like that, we can tolerate duplications, or we may extract those common utilities in a specific package...

Reply:
Note that the `agm:` is implemented in `ArbitraryPrecisionFloat`, my understanding is that you want this generalized to a collection, right? In this case, there is no conflict, just two implementations.

Reply:
Oups, case of miss-reading, you don't want a arithmetic-geometric mean, you just want indepedent arithmetic, geometric, harmonic means? Sorry for the noise.